### PR TITLE
Nessie CLI: rename `call()` to `callWithDatabaseAdapter()`

### PR DESCRIPTION
**Pull request by snazy**
+ nit: print erase-repository notification

### CODE REVIEW

File: servers/quarkus-cli/src/main/java/org/projectnessie/quarkus/cli/BaseCommand.java [hunk 1]
```diff
@@ -29,8 +30,29 @@ public abstract class BaseCommand implements Callable<Integer> {
   @Inject VersionStoreConfig versionStoreConfig;
   @Inject ServerConfig serverConfig;
 
+  @Inject Instance<DatabaseAdapter> databaseAdapterInstance;
```
Comment (dimas-b) [inline servers/quarkus-cli/src/main/java/org/projectnessie/quarkus/cli/BaseCommand.java:33]: Should `@Inject` on line 29 be removed?

Comment (snazy) [reply servers/quarkus-cli/src/main/java/org/projectnessie/quarkus/cli/BaseCommand.java:33]: Nope, that's the "quarkus specific" one

Comment (dimas-b) [reply servers/quarkus-cli/src/main/java/org/projectnessie/quarkus/cli/BaseCommand.java:33]: ... but we overwrite it from `databaseAdapterInstance.get()` in `call()` :thinking: 

What is the difference between these two adapter objects?

Comment (snazy) [reply servers/quarkus-cli/src/main/java/org/projectnessie/quarkus/cli/BaseCommand.java:33]: err, wrong line 29

**Review by dimas-b — APPROVED** [review]

**Pull request merged**